plugins/juju/04known_bugs

```diff
@@ -0,0 +1,70 @@
+#!/usr/bin/python3
+import logging
+import subprocess
+
+from common import helpers, constants
+from juju_common import (
+    JUJU_LOG_PATH
+)
+
+
+log = logging.getLogger("juju.04known-bugs")
+
+
+def detect_bug1910958():
+    """Unit fails to start complaining there are members in the relation."""
+    result = {
+        1910958: {
+            "description": ("Unit fails to start complaining there are "
+                            "members in the relation."),
+            "match": ""
+            }
+        }
+    pattern = (
+        'manifold worker returned unexpected error: failed to initialize '
+        'uniter for "[A-Za-z0-9\\-]+": cannot create relation state tracker: '
+        'cannot remove persisted state, relation \\d+ has members'
+    )
+
+    cmd = ("-r", "-P", "-H", "--include=*.log",
+           pattern, JUJU_LOG_PATH)
+    try:
+        output = helpers.grep(*cmd)
+        output = output.strip()
+        disp_text = []
+        for line in output.split("\n"):
+            if len(line) > constants.MAX_MATCH_CHARS:
+                disp_text.append(line[:constants.MAX_MATCH_CHARS] + "...")
+            else:
+                disp_text.append(line)
+
+        result[1910958]["match"] = "\n".join(disp_text)
+
+    except subprocess.CalledProcessError:
+        return None
+
+    return result
+
+
+KNOWN_ISSUES = [detect_bug1910958]
+
+
+def main():
+    issues_found = {"launchpad": []}
+    for func in KNOWN_ISSUES:
+        result = func()
+        if result:
+            issues_found["launchpad"].append(result)
+
+    if not helpers.HOTSOSYaml.master_has_plugin("juju"):
+        issues_found = {"juju": issues_found}
+        indent = 0
+    else:
+        indent = 2
+
+    return (issues_found, indent)
+
+
+if __name__ == "__main__":
+    (issues_found, indent) = main()
```

Comment:
this will still print something if issues_found is empty. also, if we want to be able to report issues from > launchpad we need a top-level key like "known-bugs"